OxfordBodleian/Juel-Jensen/BDLaethg41.xml

```diff
@@ -37,9 +37,111 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                     <msContents>
                         <summary/>
                         
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
+                        <msItem xml:id="ms_i1">
+                      <locus from="1r1" to="1r23"/>
+                            <title type="complete" ref="LIT5920JohnBeg"/>
+                           <textLang mainLang="gez"/>
+                            <incipit xml:lang="gez"><hi rend="rubric">ወንጌል፡ ዘዮሐንስ፡ ቀዳሚሁ፡ ቃል፡ ውእቱ፡ ቃል፡</hi> ወውእቱ፡ ቃል፡ ኀበ፡ 
+                            እግዚአብሔር፡ ውእቱ፡ ወእግዚአብሔር፡ ውእቱ፡ ቃል፡ ወከማሁ፡ ቀዳሚሁ፡ እምቀዲሙ፡ ኀበ፡ እግዚአብሔር፡ ውእቱ፡</incipit>
+                            <explicit xml:lang="gez">ወብርሃኑሰ፡ ዘውስተ፡ ጽልመት፡ ያበርህ፡ ወያርኢ፡ ወጽልመትኒ<supplied reason="omitted">፡</supplied>
+                                ኢይረክቦ፡ ወኢይቅርቦ፡ ወከማሁ፡ ኢይርከብዎ፡ 
+                            ወኢይቅረብ<hi rend="rubric">ዎ</hi>፡ ባርያ፡ ወለጌዎን፡ ወባርያ፡ ጸሊማን፡ ወባርያ፡ ቀይሐን፡ ወባርያ፡ ጥቁር፡ ከመ፡ ኢይቅረቡ፡ ኀበ፡
+                            ነፍ<choice><orig>ሳ</orig><corr>ሱ</corr></choice>፡ 
+                                ወሥጋ<choice><orig>ሀ</orig><corr>ሁ</corr></choice>፡ ለ<hi rend="rubric">ገብርከ፡ 
+                                    <persName ref="PRS13856Elyas">ኤልያስ፡</persName>
+                                    <persName ref="PRS13857Baselyos">ባስዮስ፡</persName> <persName ref="PRS13858MaryamS">ማርያም፡ ስና፡</persName></hi></explicit>
```

Comment:
Is in ባስዮስ፡ a ል missing in the scroll? If so, since the name is clearly ባስልዮስ፡, is perhaps `<supplied reason="omitted">ል</supplied>` necessary? (also in the next text)

Reply:
I've added sic, is it ok? The fact that this spelling occurs twice makes me wonder whether it points to an assimilation rather than a scribal omission

Reply:
`<sic/>` sounds honest enough: no speculation, just an observation